docs/event-bus/docs/013-service-programming-model.md

```diff
@@ -0,0 +1,80 @@
+| **kyma-topic** | Topic name used internally by the Kyma Event Bus to deliver messages to the subscribers via a Message Bus. |
+| **x-b3-flags** | Header used by the Zipkin tracer in Envoy. It encodes one or more options. See more on Zipkin tracing [here](https://github.com/openzipkin/b3-propagation). |
+| **x-b3-parentspanid** | Header used by the Zipkin tracer in Envoy. The **ParentSpanId** is 64-bit in length and indicates the position of the parent operation in the trace tree. When the span is the root of the trace tree, the **ParentSpanId** is absent. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation) |
+| **x-b3-sampled** | Header used by the Zipkin tracer in Envoy. When the **Sampled** flag is either not specified or set to `1`, the span is reported to the tracing system. Once **Sampled** is set to `0` or `1`, the same value should be consistently sent downstream. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). |
+| **x-b3-spanid** | Header used by the Zipkin tracer in Envoy. The **SpanId** is 64-bit in length and indicates the position of the current operation in the trace tree. The value should not be interpreted. It may or may not be derived from the value of the **TraceId**. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). |
+| **x-b3-traceid** | Header used by the Zipkin tracer in Envoy. The **TraceId** is 64-bit in length and indicates the overall ID of the trace. Every span in a trace shares this ID. See more on zipkin tracing [here](https://github.com/openzipkin/b3-propagation). |
+| **x-request-id** | Randomly generated ID which identifies the HTTP request delivering the business Event. |
+| **x-envoy-decorator-operation** | If this header is present in Ingress requests, its value overrides any locally defined operation (span) name on the server span generated by the tracing mechanism. If this header is present in an Egress response, its value overrides any locally defined operation (span) name on the client span. |
+| **x-envoy-expected-rq-timeout-ms** | Time in milliseconds in which the router expects the request to be completed. Envoy sets this header so that the upstream host receiving the request can make decisions based on the request timeout. It is set on internal requests and is either taken from the **x-envoy-upstream-rq-timeout-ms** header or from the route timeout. |
+| **x-istio-attributes** | Istio specific metadata. |
+
+## Event Payload
+
+The Event Payload is delivered as the body of the HTTP Request in JSON format. The JSON schema is available in the Service Catalog in the registered service for the remote Events.
+
+## Event Payload Example
+
+In this example, you write a service for an `order.created` Event published by the `external solution` service. The published Event schema looks as follows:
```

Comment:
``published by the `external solution` service.``  -> or maybe just: `published by the external solution.`
Does it make a difference?